CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ All notable changes to this project will be documented in this file.
 - Fix a bug where changes to ConfigMaps that are referenced in the NifiCluster spec didn't trigger a reconciliation ([#772]).
 - The operator now emits a warning (1.x.x) or errors out (2.x.x) if a deprecated or unsupported sensitive properties algorithm is used ([#799]).
 - Allow uppercase characters in domain names ([#817]).
+- NiFi v2 now exposes a metrics service (pointing to the same port as the headless service) ([#819]).
```

Comment:
As this was not a across releases thing, we could just add this to the list of PRs for "Added Listener support for NiFi"

Reply:
https://github.com/stackabletech/nifi-operator/pull/819/commits/a2226e7dd09ccbe767352376a2fc48d796503c2a